### PR TITLE
Merge develop

### PR DESCRIPTION
**Pull request by banterCZ**
Original releases/1.4.x branch was missing some important commits, especially war name with postfix `-wultra`

### CODE REVIEW

File: enrollment-server-onboarding/pom.xml [hunk 1]
```diff
@@ -21,8 +21,8 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>enrollment-server-onboarding</artifactId>
-    <name>enrollment-server-onboarding</name>
+    <artifactId>enrollment-server-onboarding-wultra</artifactId>
```
Comment (banterCZ) [inline enrollment-server-onboarding/pom.xml:24]: ❗

File: enrollment-server/pom.xml [hunk 2]
```diff
@@ -21,10 +21,10 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <name>enrollment-server</name>
+    <name>enrollment-server-wultra</name>
```
Comment (banterCZ) [inline enrollment-server/pom.xml:24]: ❗

**Comment by zcgandcomp** [comment]
sorry it was not done properly and commits and dependencies after 1.4.2 are here (e.g. shedlock) this we should not merge

**Comment by banterCZ** [comment]
> sorry it was not done properly and commits and dependencies after 1.4.2 are here (e.g. shedlock) this we should not merge

I think that we may keep the newer shedlock (or revert if needed). Anything else?

**Review by romanstrobl — APPROVED** [review]
Looks OK.

**Comment by banterCZ** [comment]
Replaced by #96

**Pull request closed without merging**